### PR TITLE
DDF-1944 Removes DynamicImport-Package directives in bundles.

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
This PR updates pom files that had been using `DynamicImport-Package`, removing that directive and importing the packages directly.

#### Who is reviewing it?
@kcwire @pklinef @stustison 

#### How should this be tested?
1. The application should be built and itests run. 
2. The application should be deployed and the installation process should be run.
3. Finally, the `security-sts-ldaplogin` bundle (or the feature that contains it) should be installed to ensure that it works correctly.

#### Any background context you want to provide?
N/A

#### What are the relevant tickets?
[DDF-1944](https://codice.atlassian.net/browse/DDF-1944)

#### Screenshots (if appropriate)
N/A

#### Checklist:
- [ ] Documentation Updated N/A
- [ ] Update / Add Unit Tests N/A
- [ ] Update / Add Integration Tests N/A

### CODE REVIEW

**Comment by brendan-hofmann** [comment]
Reading the provided link in DDF-1944 implies that this is a performance issue. Is there any noticeable gain with these changes, or is this more of a best practices issue for us?

**Comment by pklinef** [comment]
:+1:

File: catalog/opensearch/catalog-opensearch-endpoint/pom.xml [hunk 1]
```diff
@@ -141,6 +141,7 @@
                         <Embed-Dependency>common-system</Embed-Dependency>
                         <Import-Package>
                             META-INF.cxf,
+                            javax.ws.rs.*,
```
Comment (kcwire) [inline catalog/opensearch/catalog-opensearch-endpoint/pom.xml]: did you have to add this? just wondering if maven-bundle-plugin would have figured this out on its own.

**Comment by kcwire** [comment]
:shipit:

File: catalog/opensearch/catalog-opensearch-endpoint/pom.xml (hunk 1 above)
Comment (coyotesqrl) [inline catalog/opensearch/catalog-opensearch-endpoint/pom.xml]: I'm not completely sure, to be honest. When I went through the poms I brute-forced the dynamic packages into the import directives one pom after the other. The `*` in this one probably would have picked it up. I can check on it later.

Comment (coyotesqrl) [inline catalog/opensearch/catalog-opensearch-endpoint/pom.xml]: It was unneeded. I removed and rebased with master.

**Comment by ryeats** [comment]
:+1:  heroed

**Pull request merged**